package.json

```diff
@@ -21,7 +21,7 @@
     "integ:centerpoint": "python3 lib/osml-model-runner-test/bin/process_image.py --image small --model centerpoint",
     "integ:flood": "python3 lib/osml-model-runner-test/bin/process_image.py --image large --model flood",
     "integ:meta": "python3 lib/osml-model-runner-test/bin/process_image.py --image meta --model centerpoint",
-    "integ:tile-server": "aws lambda invoke --region $AWS_DEFAULT_REGION --function-name TSTestRunner --payload '{\"test_type\": \"integ\", \"source_image_key\": \"small.tif\"}' --log-type Tail /dev/null --cli-read-timeout 0 --query 'LogResult' --output text | base64 -di",
+    "integ:tile-server": "aws lambda invoke --region $AWS_DEFAULT_REGION --function-name TSTestRunner --payload '{\"image_uri\": \"s3://osml-test-images-409719124294/small.tif\"}' --log-type Tail /dev/null --cli-read-timeout 0 --query 'LogResult' --output text | base64 -di",
     "load:tile-server": "aws lambda invoke --region $AWS_DEFAULT_REGION --function-name TSTestRunner --payload '{\"test_type\": \"load\", \"locust_image_keys\": \"small.tif,meta.ntf,sicd-capella-chip.ntf,sicd-interferometric-hh.nitf\", \"locust_headless\": \"true\", \"locust_run_time\": \"5m\", \"locust_users\": \"1\", \"locust_spawn_rate\": \"1\"}' --log-type Tail /dev/null --cli-read-timeout 0 --query 'LogResult' --output text | base64 -di",
```

Comment:
If we are no longer deploying the load test to lambda we should remove this script